prody/proteins/interactions.py

```diff
@@ -578,7 +579,9 @@ def calcPiStacking(atoms, **kwargs):
     aromatic_dic = {'TRP':'noh and not backbone and not name CB NE1 CD1 CG',
                 'PHE':'noh and not backbone and not name CB',
                 'TYR':'noh and not backbone and not name CB and not name OH',
-                'HIS':'noh and not backbone and not name CB'}
+                'HIS':'noh and not backbone and not name CB',
```

Comment:
Do we need to handle alternative histidine for salt bridges too? I guess not HSE and HSD because these are single protonated neutral but the other one

Reply:
I was thinking about that too. HSP is protonated in CHARMM ff. We might remove HSE and HSD. Unless we would like to keep them in case during the simulation we would observe some reorganization and then HSD/HSE will come closer to negatively charged residue. Then the prediction of HIS to be HSE might be wrong. Is it sufficient reason to keep those?